controllers/v1/project/templates.js

```diff
@@ -666,12 +667,16 @@ module.exports = class ProjectTemplates extends Abstract {
 
        async details(req) {
         return new Promise(async (resolve, reject) => {
+            
             try {
-
+                (req.params._id) ? req.params._id=req.params._id : req.params._id="";
```

Comment:
@VISHNUDAS-tunerlabs remove this and handle it while calling the helper function